features/largeVideo/components/LargeVideo.js

```diff
@@ -63,9 +64,14 @@ class LargeVideo extends Component {
             this.props.dispatch(participantSelected(activeParticipant.id));
         }
 
+        let mirrored = videoTrack &&
+            videoTrack.isLocal() &&
+            videoTrack.videoType === 'camera';
```

Comment:
@legastero, please think about the above & let's talk about our ideas during tomorrow's meeting.